tidb-cloud/migration-job.md

```diff
@@ -0,0 +1,115 @@
+
+To solve this problem, TiDB Cloud provides the Migration Job feature, which supports full migration and incremental migration, allowing you to migrate your business from the upstream database to TiDB Cloud within a short downtime window.
+
+A migration job supports the following data sources:
+
+- MySQL 5.6-8.0 on premises or on a cloud
+- AWS Aurora MySQL 5.6 and 5.7
+- AWS RDS MySQL 5.7
+
+A migration job supports the following network types:
+
+- Public IP
+- VPC peering
+- Private Link
+
+A migration job supports data migration within the same region and cross regions.
+
+## Limitations
+
+- The system databases of `mysql`, `information_schema`, `information_schema` and `sys` will be filtered out during migration (even when you migrate the entire upstream MySQL instance), and will not be migrated to TiDB Cloud.
+
+- When you delete a cluster, all migration jobs in that cluster are automatically deleted, and the deleted migration jobs are not recoverable.
+
+## Prerequisites
+
+The username you use for the upstream database must have the following privileges:
+
+| Privilege | Scope |
+|:----|:----|
+| `SELECT` | Tables |
+| `RELOAD` | Global |
+| `REPLICATION SLAVE` | Global |
+| `REPLICATION CLIENT` | Global |
+| `PROCESS` | Global |
+
+For example, if you need to migrate the data from `db1` to TiDB Cloud, execute the following `GRANT` statement:
+
+```sql
+GRANT RELOAD, REPLICATION SLAVE, REPLICATION CLIENT ON *.* TO 'your_user'@'your_wildcard_of_host'
+GRANT SELECT ON db1.* TO 'your_user'@'your_wildcard_of_host';
+```
+
+If you also need to migrate the data from other databases to TiDB Cloud, make sure the same privileges are granted to the user of the respective databases.
+
+## Step 1: Go to the **Data Migration** page
+
+1. Log in to the [TiDB Cloud console](https://tidbcloud.com/console/clusters). Navigate to the **Clusters** page for your project.
+
+2. On the **Clusters** page, click the name of your cluster, and click the **Data Migration** tab. The **Data Migration** page is displayed.
+
+3. On the **Data Migration** page, click **Create Migration Job**. The **Create Migration Job** page is displayed.
+
+## Step 2: Configure the source and target connection
+
+On the **Create Migration Job** page, configure the source and target connection.
+
+1. Create a job name. It must consist of numbers and letters with less than 60 characters.
+
+2. Fill in the source connection profile.
+
+   - **Data source**: the data source type. Currently, it supports MySQL, AWS Aurora MySQL, and AWS RDS MySQL.
+   - **Region**: the region of the data source. If the source database is a self-built, the parameter is empty.
+   - **Connectivity method**: the connectivity method of the data source. Currently, it supports Public IP, VPC peering, and Private Link.
+   - **Hostname or IP address**: the hostname or IP address of the data source.
```

Comment:
for private link user need to input `service name`, and need user to accept it in their aws account

Reply:
OK added